src/components/Sum.jsx

```diff
@@ -1 +1,22 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({
+  a = 0,
+  b = 0,
+}) {
```

Comment:
```suggestion
export function Sum({ a = 0, b = 0 }) {
```
You can leave it in one line as there are < 3 props and the line is quite short

Reply:
Fixed